modules/network/main.tf

```diff
@@ -0,0 +1,177 @@
+resource "aws_vpc" "main" {
```

Comment:
Building everything from scratch is good for learning, but using the shared modules is also good, as it's [recommended](https://nimblehq.co/compass/development/code-conventions/terraform/#use-shared-modules) by our Compass. But I'm ok as we're trying to learn this :D

Reply:
@hoangmirs I forgot we have this page on compass, my bad! 

Yup it had its benefits on the IC initially, I will have to think when refactoring according to our standards I do this too